__tests__/validateAuth.tsx

```diff
@@ -0,0 +1,35 @@
+import {getBooks} from 'src/services/booksService';
+import api from 'src/config/api';
+import {validateEmail, validatePassword} from 'src/utils/validations';
+
+describe('validate Password', () => {
+  test('invalid password', () => {
+    expect(validatePassword('asd')).toBe('Contraseña invalida');
+  });
+  test('valid password', () => {
+    expect(validatePassword('wowowo1134')).toBe(undefined);
+  });
+  test('required password', () => {
+    expect(validatePassword('')).toBe('Contraseña requerida');
+  });
+});
+
+describe('validate email', () => {
+  test('invalid password', () => {
```

Comment:
You are repeated the names of the tests here, if you are going to copy paste, do it with style ;)